app/src/components/blocks/_lloydGeorge/lloydGeorgeViewRecordStage/LloydGeorgeViewRecordStage.test.tsx

```diff
@@ -143,6 +143,48 @@ describe('LloydGeorgeViewRecordStage', () => {
         });
     });
 
+    it('exits full screen when browser back button is pressed', async () => {
+        renderComponent();
+
+        act(() => {
+            userEvent.click(screen.getByText('View in full screen'));
+        });
+
+        await waitFor(() => {
+            expect(screen.getByText('Exit full screen')).toBeInTheDocument();
+        });
+
+        act(() => {
+            window.dispatchEvent(new PopStateEvent('popstate'));
+        });
+
+        await waitFor(() => {
+            expect(screen.getByText('Lloyd George record')).toBeInTheDocument();
+        });
+    });
+
+    it('updates history correctly when exiting full screen using the UI back button', async () => {
```

Comment:
The test on line 124 mentions "Exit full screen link" whereas line 166 mentions "UI back button". As far as I'm aware, they both are referring to the same button. Could we name these identically for consistency?

Reply:
ah yes. Good catch. Thanks for that. will make them consistent.